scripts/util.ts

```diff
@@ -88,3 +88,12 @@ export async function addTokens(
   // Return token information
   return tokens;
 }
+
+export function parseArgs(): string[] {
+  const args = process.argv.slice(4);
+  const index = args.indexOf("--network");
+  if (index > -1) {
+    args.splice(index, 2);
+  }
+  return args;
+}
```

Comment:
I feel like this should also move to `invoke_view_function.ts` since it is very specific to that script.

Reply:
Except it is used in two different scripts. see `wait_seconds.ts` below.

Reply:
Ah makes sense! Sorry!